SPECS/ca-certificates-prebuilt/ca-certificates-prebuilt.spec

```diff
@@ -0,0 +1,42 @@
+Summary:        Prebuilt version of ca-certificates package
+Name:           ca-certificates-prebuilt
+
+Version:        20200720
+Release:        1%{?dist}
+License:        MIT
+
+URL:            https://hg.mozilla.org
+Group:          System Environment/Security
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+
+BuildArch:      noarch
+
+BuildRequires:  ca-certificates-base
+Conflicts:      ca-certificates-base
+Conflicts:      ca-certificates
+
+%description
+Prebuilt version of the ca-certificates-base package with no runtime dependencies.
+
+%prep -q
+
+%build
+
+%install
+
+mkdir -p $RPM_BUILD_ROOT%{_datadir}/pki/ca-trust-legacy/
+mkdir -p $RPM_BUILD_ROOT%{_sysconfdir}/pki/
+
+install -p -m 644 %{_datadir}/pki/ca-trust-legacy/* $RPM_BUILD_ROOT%{_datadir}/pki/ca-trust-legacy/
+install -p -m 644 %{_datadir}/pki/ca-trust-source/* $RPM_BUILD_ROOT%{_datadir}/pki/ca-trust-legacy/
+cp -r %{_sysconfdir}/pki/* $RPM_BUILD_ROOT%{_sysconfdir}/pki/
+
+%files
```

Comment:
To be more aligned with the regular `ca-certificates` packages, let's move this to a `*-base` subpackage. Otherwise the naming is getting a bit confusing. Especially if at one point we'd need to add more prebuilt certificates separately for each of the regular ones and we'll end up with a conflict with `ca-certificates` providing Mozilla certs. Since we don't need the Mozilla certs right now, the default `ca-certificates-prebuilt` subpackage won't be needed.